### ISSUE TITLE
Android: Images in Listview disappear when going back to Window

### ISSUE THREAD

**Issue by m1ga** [AndreaVitale/av.imageview#104]
_I think we had this issue in older releases before but I can't remember:_

Window A: av.ImageView inside a listview. I click the item and open a new window B (also has a av.imageview)
Then I close window B and go back to win A. The images disappear and I have to scroll down and up again to see them.

```
<ListView id="list_feed" defaultItemTemplate="default" onItemclick="onClickItem">
	<Templates>
		<ItemTemplate name="default">
			<View class="view_all">
				<ImageView class="img_feed" ns="AvImageview"/>
			</View>
		</ItemTemplate>
	</Templates>
	<ListSection/>
</ListView>
```
(removed unnecessary views etc)

When I use a normal ImageView it works fine. Here is a comparison video:
https://www.migaweb.de/avimageview.mp4
first half (squashed images) are normal ImageViews, starting at 0:07 I use av.imageviews

Ti 9.2.2, av.imageview 5.0.1 (and 5.0.2)

**Comment by arifje** [comment]
Experiencing the same issue here!

**Comment by macasfaj** [comment]
Yes, that issue is there since 4.0.0 :(

**Comment by m1ga** [comment]
Do you use `updateItemAt` before opening the 2nd window? 
I run that twice (showing/hiding the heart icon) and that breaks the imageview somehow! If I open the new window with a setTimeout or without the `udpateItemAt` block it works fine!

*edit*
even a timeout works already:
```
function onClickItem(e) {
	var idx = e.itemIndex;
	var item = $.list_feed.sections[0].getItemAt(idx);
	item.heart.opacity = 1;
	$.list_feed.sections[0].updateItemAt(idx, item);
	setTimeout(function() {
		item.heart.opacity = 0;
		$.list_feed.sections[0].updateItemAt(idx, item);
	}, 500);

	// with this timeout it works again!
	setTimeout(function() {
		var detail = Alloy.createController("/detail");
		Alloy.Globals.navWin.openWindow(detail.getView());
	}, 1000);
}
```

**Comment by prashantsaini1** [comment]
Could you guys please give a try to the attached module in this PR : https://github.com/AndreaVitale/av.imageview/pull/108

**Comment by m1ga** [comment]
yes, that fixed it for me! Images are still there when I close my window - even without the timeout workaround in my last message! :+1:

**Comment by prashantsaini1** [comment]
Good to hear. It was then activity leakage issue.

**Comment by AndreaVitale** [comment]
Merged the #108 ! Thanks guys!
